sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -659,6 +659,16 @@ public partial class RetryOptions
         public System.TimeSpan NetworkTimeout { get { throw null; } set { } }
     }
     public delegate System.Threading.Tasks.Task SyncAsyncEventHandler<T>(T e) where T : Azure.SyncAsyncEventArgs;
+    public partial class TelemetryDetails
```

Comment:
Do we really like the rename?

Reply:
I like it more than anything with `UserAgent` in the name.  If you hate the `Details` suffix, we could call it `TelemetryData`, `ClientTelemetry`, `TelemetryBuilder`, etc.